.evergreen/.evg.yml

```diff
@@ -1069,8 +1069,8 @@ buildvariants:
     - name: "plain-auth-test"
 
 - matrix_name: "aws-auth-test"
-  matrix_spec: { jdk: "jdk11" }
-  display_name: "MONGODB-AWS Auth test"
+  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "jdk11", version: ["4.4", "latest"], os: "ubuntu" }
```

Comment:
Let's just on jdk8 as well, since that's our minimum supported version.

Reply:
Done. Also, test are failing for jdk8 on both latest and 4.4

Reply:
Same error for both?